workflow/ttlcontroller/ttlcontroller.go

```diff
@@ -141,15 +141,6 @@ func (c *Controller) deleteWorkflow(ctx context.Context, key string) error {
 	return nil
 }
 
-// if the workflow both has a TTL and is expired
-func (c *Controller) ttlExpired(wf *wfv1.Workflow) bool {
```

Comment:
I think we should remove this function(at least from the controller) since it is only used in the test and not part of the functional code of the controller atm.